lang/en/block_equella_links.php

```diff
@@ -36,3 +36,4 @@
 
 $string['configincludetaggeditems'] = 'Content tagged to this course (inc. eReadings)';
 $string['configxmlpath'] = 'XML Path (your EQ admin can provide)';
+$string['privacy:metadata'] = "The LINKS block does not store any data - it uses shared secret to interact with openEQUELLA.";

```

Comment:
I note this text is the same on all of them, but I'm not sure of the capitalisation. Could it just be:

```suggestion
$string['privacy:metadata'] = "This block does not store any data - it uses a shared secret to interact with openEQUELLA.";

```

If we do change it here, we should do the same in the others.